crates/primitives-traits/src/node.rs

```diff
@@ -15,7 +15,7 @@ pub trait NodePrimitives:
     /// Block body primitive.
     type BlockBody: FullBlockBody<Transaction = Self::SignedTx, OmmerHeader = Self::BlockHeader>;
     /// Signed version of the transaction type.
-    type SignedTx: FullSignedTx;
+    type SignedTx: FullSignedTx + MaybeSerdeBincodeCompat;
```

Comment:
can we move this to `FullSignedTx` instead

like:

https://github.com/paradigmxyz/reth/blob/77bd2115fb8084c6b336446959a07a62a53b4d27/crates/primitives-traits/src/block/body.rs#L13-L15

Reply:
Done